edx_solutions_api_integration/mobile_api/views.py

```diff
@@ -172,6 +173,14 @@ def _get_course_average_grade(self, course_key):
         Note: For performance reasons, we use the cached gradebook data here.
         Once persistent grades are enabled on the solutions fork, we'll use CourseGradeFactory instead.
         """
-        return StudentGradebook.course_grade_avg(
-            course_key
-        )
+        cache_key = 'course_grade_avg_{}'.format(course_key)
+        cache_ttl = 60 * 30  # 30 minutes
```

Comment:
I think we should set this `cache_ttl ` in some settings where we can change cache time at any time if required.

Reply:
There's no settings module in this package; also cache settings are already placed in some modules. e.g;
https://github.com/edx-solutions/api-integration/blob/master/edx_solutions_api_integration/utils.py#L35